control-plane/connect-inject/common/common.go

```diff
@@ -73,6 +73,20 @@ func PortValue(pod corev1.Pod, value string) (int32, error) {
 	return int32(raw), err
 }
 
+// WorkloadPortName returns the container port's name if it has one, and if not, constructs a name from the port number
+// and adds a constant prefix. The port name must be 1-15 characters and must have at least 1 alpha character.
+func WorkloadPortName(port *corev1.ContainerPort) string {
```

Comment:
can we add some unit tests for this function?

Reply:
ah yep i forgot to add unit tests on this side after refactoring